offload/DeviceRTL/src/Tasking.cpp

```diff
@@ -20,16 +20,14 @@
 
 using namespace ompx;
 
-#pragma omp begin declare target device_type(nohost)
-
 extern "C" {
 
 TaskDescriptorTy *__kmpc_omp_task_alloc(IdentTy *, int32_t, int32_t,
                                         size_t TaskSizeInclPrivateValues,
                                         size_t SharedValuesSize,
                                         TaskFnTy TaskFn) {
   auto TaskSizeInclPrivateValuesPadded =
-      utils::roundUp(TaskSizeInclPrivateValues, uint64_t(sizeof(void *)));
+      utils::roundUp(TaskSizeInclPrivateValues, size_t(sizeof(void *)));
```

Comment:
```suggestion
      utils::roundUp(TaskSizeInclPrivateValues, sizeof(void *));
```
`sizeof`, [by defintion](https://en.cppreference.com/w/cpp/language/sizeof), returns `size_t`